lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
For postgres the serialization error with be returned when you attempt to commit. But retrying a commit will do nothing in that case, you need to rerun the sequence in a new transaction.

So if Sequelize would catch the serialization error on the COMMIT query handler and keep retrying, that would be wrong :)